include/swift/AST/Decl.h

```diff
@@ -3125,9 +3125,11 @@ class NominalTypeDecl : public GenericTypeDecl, public IterableDeclContext {
   /// overridden declarations been removed.
   ///
   /// \param ignoreNewExtensions Whether to avoid loading any new extension.
+  /// \param includeAttrImplements Whether to include @_implements members.
   /// Used by the module loader to break recursion.
   TinyPtrVector<ValueDecl *> lookupDirect(DeclName name,
-                                          bool ignoreNewExtensions = false);
+                                          bool ignoreNewExtensions = false,
+                                          bool includeAttrImplements = false);
```

Comment:
It's not critical, but it might be nice for this to be an `OptionSet`. Those multiple `bool` parameters become confusing in a language without argument labels :(

Reply:
fixed